ls1intum/Artemis
# Bugfix/quiz/duration selector

### Checklist
- [x] I tested *all* changes and *all* related features with different users (tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context


When using the duration selector during the quiz creation a strange behaviour was found. The issue is described in #1391
### Description

The problem is that the input value was not updated correctly even though the model had the right value.
- Added `detectChanges()`before updating duration values
- Bring ngModelChange and ngModel into the right oder (see following issue: https://github.com/angular/angular/issues/11234)
- Removed `(blur)` as `(ngModelChange)` is already getting the changes

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Management
3. Go to exercises of a Course
4. Create a new Quiz
5. Use duration selector
5a. When being at seconds 11:59 and you click up, it should go to 12:00
5b. When being at seconds 12:00 and you click down, it should go to 11:59
5c. When being at 600:00 and you click up nothing changes, when you click down it goes to 599:59
5d. When being at 00:00 and you click down nothing changes, when you click up it goes to 00:01

### Screenshots



Test clicking up to 60 seconds and down to -1 seconds:

![video1](https://user-images.githubusercontent.com/41108487/83264219-123cf900-a1c0-11ea-8708-a02a653f6409.gif)

Test min and max value:

![video2](https://user-images.githubusercontent.com/41108487/83264304-2ed93100-a1c0-11ea-8e88-60c401767a27.gif)
![video3](https://user-images.githubusercontent.com/41108487/83264243-19fc9d80-a1c0-11ea-8a6e-b6210758004a.gif)

Delete input values and reenter values:
![video4](https://user-images.githubusercontent.com/41108487/83455432-2a31a880-a45e-11ea-9300-e641f6ce4461.gif)
